frontend/src/components/dashboard/create/telegram/TelegramDetail.tsx

```diff
@@ -80,8 +80,10 @@ const TelegramDetail = ({
         <>
           <h2>Your campaign has been sent!</h2>
           <p>
-            Some messages may have failed to send. You can retry these by
-            clicking on Retry.{' '}
+            A retry button will appear if some messages had an error while
+            sending. You can click on retry to try sending the message(s) again.
+            An export button will appear for you to download a list of failed
+            deliveries with the recipient’s mobile number.
```

Comment:
Hmm, I think it's still not obvious to users that: 

1. Exports are only available 5 mins after sending has completed
2. The export isn't on this page but rather on the campaigns dashboard

Perhaps we can change the wording to something along these lines? 

> An export button will be available on the Campaigns dashboard (link to /campaigns) 5 minutes after sending has completed for you to download a list of failed deliveries with the recipient’s mobile number.

Reply:
Agree on the wording can be further improved. The export button actually appears after 5 minutes on the details page. But I think we can make this more obvious. Since this message is also used across all the other campaign types, I'll let Sarah know and we can potentially fix this all together in develop later on.